presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestingSqlServerServer.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.sqlserver;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.prestosql.testing.TemporaryDockerContainer;
+import net.jodah.failsafe.Failsafe;
+import net.jodah.failsafe.RetryPolicy;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import java.time.Duration;
+
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static java.time.temporal.ChronoUnit.SECONDS;
+
+/**
+ * This suite is required to run any Oracle Test as it provides docker container with Oracle.
+ */
+public final class TestingSqlServerServer
+{
+    private static final Logger LOG = Logger.get(TestingSqlServerServer.class);
+
+    public static final String JDBC_URL = "jdbc:sqlserver://localhost";
+    public static final String USER = "sa";
+    public static final String PASSWORD = "SQLServerPass1";
+
+    private static TemporaryDockerContainer dockerContainer;
+
+    @BeforeSuite
+    public void setupTestingOracleServer()
+            throws Exception
+    {
+        this.dockerContainer = new TemporaryDockerContainer(
+                "microsoft/mssql-server-linux:2017-CU6",
+                "sqlserver-master",
+                ImmutableList.of(1433),
+                ImmutableMap.of(
+                        "ACCEPT_EULA", "Y",
+                        "SA_PASSWORD", "SQLServerPass1"));
+
+        RetryPolicy<Object> retryPolicy = new RetryPolicy<>()
+                .withMaxDuration(Duration.of(5, MINUTES))
+                .withMaxAttempts(Integer.MAX_VALUE) // limited by MaxDuration
```

Comment:
What would be the default if you don't set this?

Reply:
3